Program.cs

```diff
@@ -1,100 +1,116 @@
 ﻿using System;
+using System.ComponentModel;
+using System.Globalization;
 
 namespace Console_Application
 {
     internal class Program
     {
         static void Main(string[] args)
         {
-            // We are asking the user to choose an option.
-            Console.WriteLine("Choose between options 1, 2, 3, and 4." + "\n");
-            int userOptions = Convert.ToInt32(Console.ReadLine());
-
-            // Using switch case to output the correct option
-            switch (userOptions)
+            bool isRunning = true;
+            while (isRunning)
             {
-                case 1:
-                    Console.WriteLine("\n");
-                    // Request the user to input data.
-                    Console.WriteLine("Input a word or sentence." + "\n");
-                    // The data is stored in w.
-                    string input = Console.ReadLine();
-                    Console.WriteLine("\n");
-                    // Giving the user options on casing.
-                    Console.WriteLine("Do you want your input to be upper or lower case?");
-                    Console.WriteLine("Indicate by typing (u) for upper or (l) for lower." + "\n");
-                    // The users option is stored in "options".
-                    string options = Console.ReadLine();
+                // We are asking the user to choose an option.
+                Console.WriteLine("Choose between options 1, 2, 3, and 4.\n");
+
+                // Making sure the user can only input interger values.
+                if (!int.TryParse(Console.ReadLine(), out int userOptions))
+                {
+                    Console.WriteLine("Invalid input. Read please!");
```

Comment:
Ahahhan this is not user friendly lmaooo

Reply:
Also, if you have the continue the code keeps running event if the option is invalid no?

Reply:
Lmaoo, I'll update it.

The code restarts if an invalid option is inputted and the user will have to choose option 4 to exit it.